src/keywordPreservation.ts

```diff
@@ -173,14 +177,37 @@ export const updateAssetsByAddress = (
 // preserveKeywords is the function that ultimately gets executed during export
 // to attempt to preserve keywords (ex: ##KEYWORD##) in local configuration files
 // from getting overwritten by remote values during export.
-export const preserveKeywords = (
-  localAssets: object,
-  remoteAssets: object,
-  keywordMappings: KeywordMappings
-): object => {
+export const preserveKeywords = ({
+  localAssets,
+  remoteAssets,
+  keywordMappings,
+  auth0Handlers,
+}: {
+  localAssets: object;
+  remoteAssets: object;
+  keywordMappings: KeywordMappings;
+  auth0Handlers: Pick<APIHandler, 'id' | 'identifiers' | 'type'>[];
+}): object => {
```

Comment:
he addition of the Auth0 handlers argument made this function signature very crowded. Instead, I re-implemented this function to accept a single object of these arguments. This forces the naming of these arguments to be clearer and ordering to not matter.